packages/core/src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -15,6 +14,7 @@ import * as vscode from 'vscode'
 import globals from '../../extensionGlobals'
 import { SamCliSettings } from './samCliSettings'
 import { addTelemetryEnvVar, collectSamErrors, SamCliError } from './samCliInvokerUtils'
+import { fs } from '../..'
```

Comment:
Avoid importing from `index.ts` files in core, it can cause circular dependencies (e.g. we export this file through that `index.ts` directly or indirectly). Instead import directly from the file.

Reply:
Out of curiosity, what purpose does `index.js` serve if its not supposed to be imported from?

Reply:
> Is it just for exporting to external modules?

Yes, it's for the `packages/amazon/` and `packages/toolkit/` modules, which depend on `packages/core/`.